### ISSUE TITLE
求救bubbliiing大神：Cannot iterate over a shape with unknown rank.

### ISSUE THREAD

**Issue by wt616** [bubbliiiing/mask-rcnn-keras#20]
tensorflow：CPU版1.31.1
Keras：2.3.1（如果低于这个版本会有一堆冲突）
如下图按要求布置好训练文件到文件夹后，在compile的时候，会报这个错误。在这个项目的其他issue里也看到不少人遇到，目前没看到解决办法
![image](https://user-images.githubusercontent.com/8064790/138639029-ffc88d6f-296f-4730-b572-59bfcd3ebd6f.png)
